.travis.yml

```diff
@@ -121,41 +120,41 @@ install:
       # dependencies
       brew update > /dev/null
       brew outdated "pkg-config" || brew upgrade "pkg-config"
-      brew install colormake ccache zlib qt
+      brew install colormake ccache zlib qt libtorrent-rasterbar
       PATH="/usr/local/opt/ccache/libexec:$PATH"
       brew link --force zlib qt
 
-      wget https://builds.shiki.hu/homebrew/version
-      if ! cmp --quiet "version" "$HOME/hombebrew_cache/version" ; then
-        echo "Cached files are different from server. Downloading new ones."
-        # First delete old files
-        rm -r "$HOME/hombebrew_cache"
-        mkdir "$HOME/hombebrew_cache"
-        cp "version" $HOME/hombebrew_cache
-        cd "$HOME/hombebrew_cache"
-        wget https://builds.shiki.hu/homebrew/libtorrent-rasterbar.rb
-        wget https://builds.shiki.hu/homebrew/5146d2df7e48f321511273fb9829ebdc3a6dc519f9ef36a344a343ae524c3ae6--libtorrent-rasterbar-1.1.9+git20180812.0bcf6cef23+patched-configure.el_capitan.bottle.tar.gz
+      if [ "$build_system" = "cmake" ]; then
+        brew outdated cmake || brew upgrade cmake
+        brew install ninja
+
+        sudo ln -s /usr/local/opt/qt/mkspecs /usr/local/mkspecs
+        sudo ln -s /usr/local/opt/qt/plugins /usr/local/plugins
       fi
 
+      #wget https://builds.shiki.hu/homebrew/version
```

Comment:
Commenting out in a VCS controlled file? Does it make sense at all?

Reply:
@sledgehammer999 
Are you OK with removing this part?

Reply:
>@sledgehammer999 isn't interested in CMake build.

This part is common path for macOS.

I'll remove it if no reply.